src/rest.ts

```diff
@@ -25,16 +28,17 @@ const CR = '\r'.charCodeAt(0); // always 13, but avoid magic constant
 
 const PODMAN_SYSTEM_ADDRESS = "/run/podman/podman.sock";
 
-/* uid: null for logged in session user, otherwise standard Unix user ID
- * Return { path, superuser } */
-function getAddress(uid) {
+export type Uid = number | null; // standard Unix UID or null for logged in session user
+
+// FIXME: export SuperuserMode in cockpit.d.ts, and use it here
```

Comment:
You can do things like `cockpit.ChannelOptions["superuser"]` to get at that type even without exporting it.